src/utils/__tests__/permissions.test.js

```diff
@@ -0,0 +1,43 @@
+import {
+  hasDeleteArticlePermission,
+  hasCreateArticlePermission,
+  hasEditArticlePermission,
+} from '../permissions';
+
+const mockedUser = {
+  _id: 'test_user_id_1',
```

Comment:
i think when you mock user, u need to include all fields, how user should look like